apps/site/lib/site_web/templates/error/error_layout.html.eex

```diff
@@ -0,0 +1,42 @@
+
+<!-- Default -->
+<div class="container error-page average-sizes">
```

Comment:
Unless I'm mistaken it looks like the only difference between Default and XL is the moving of `<div class="heading-1 error-title"> <%= @error_title %></div>`. Instead of using a class to toggle one entire `.error-page` vs another (loading the SVG and everything else twice and whatnot), can we limit that strategy to the .error-title?

I'm thinking, you can have two instances of `<div class="heading-1 error-title"> <%= @error_title %></div>`, the one at this level that only shows on average sizes, and then the other one inside the `.error-details-container` that shows only at the largest breakpoint.

Reply:
Oh, and you wouldn't even need an .xl-only class! You could select it via `.error-details-container .error-title {}` vs `.error-pre-title + .error-title {}` (just as an example)